src/decoders/cfa.rs

```diff
@@ -140,10 +141,9 @@ impl CFA {
     }
 
     CFA {
-      name: name,
-      pattern: pattern,
-      width: self.width,
-      height: self.height,
+      name,
+      pattern,
+      .. *self
```

Comment:
I don't particularly like this, makes the code opaque for no reason. It's only 2 simple fields.